apps/igloo-kit-cli/src/cli/webserver.rs

```diff
@@ -31,11 +38,30 @@ async fn handler(req: Request<Body>, route_table: Arc<Mutex<HashMap::<PathBuf, R
                 });
 
                 let bytes = hyper::body::to_bytes(req.into_body()).await.unwrap();
-                let body = String::from_utf8(bytes.to_vec()).unwrap();       
+                let body = String::from_utf8(bytes.to_vec()).unwrap();
 
-                return Ok(Response::builder()
-                    .status(StatusCode::FOUND)
-                    .body(body)?)
+                let guard = route_table.lock().await;
+                let topic_name = &guard.get(&route).unwrap().table_name;
+
+                let res = configured_producer.lock().await.producer.send(
```

Comment:
Damn that's a lot of locks. I get why it needs to be this way for dev. For prod we should assume the routing is immutable. Not sure if that changes how you would write the code

Reply:
Good call. We should have two paths here. One for dev mode and one for prod.